src/backend/TrafficCourts/Citizen.Service/Startup.cs

```diff
@@ -2,19 +2,20 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
 using NodaTime;
+using OpenTelemetry.Trace;
 using Serilog;
 using System.Configuration;
+using System.Reflection;
 using TrafficCourts.Citizen.Service.Configuration;
 using TrafficCourts.Citizen.Service.Services;
 using TrafficCourts.Citizen.Service.Validators;
+using TrafficCourts.Citizen.Service.Mappings;
+using TrafficCourts.Citizen.Service.Services.Impl;
 using TrafficCourts.Common;
 using TrafficCourts.Common.Configuration;
 using TrafficCourts.Messaging;
-using TrafficCourts.Citizen.Service.Services.Impl;
-using System.Reflection;
-using OpenTelemetry.Trace;
-using TrafficCourts.Citizen.Service.Mappings;
-
+using TicketStorageType = TrafficCourts.Citizen.Service.Configuration.TicketStorageType;

```

Comment:
Is this still in here? Looks like it is in `TrafficCourts.Common.Configuration`

Reply:
Yeah, this line was needed to compile the Citizen Api.  The citizen-api is not using common configuration yet, but rather it's own file persistence setup.  For this PR, I just took was citizen was doing and copied it into Common so it could be used by the staff-api.  In a future PR, I think someone should fix citizen api to use common instead.